simulations/vip-502/arbitrumone.ts

```diff
@@ -0,0 +1,19 @@
+import { NETWORK_ADDRESSES } from "src/networkAddresses";
+import { LzChainId } from "src/types";
+import { forking, testForkedNetworkVipCommands } from "src/vip-framework";
+
+import vip407 from "../../vips/vip-407/bscmainnet";
```

Comment:
Remote commands of VIP-407 have been already executed. We could remove this step from every simulation